frontend/components/notifications/management/NotificationsManagementDashboards.vue

```diff
@@ -56,6 +118,26 @@ const onEdit = (col: 'delete' | 'subscriptions' | 'webhook' | 'networks', row: N
   }
 }
 
+async function saveUserSettings (settingsAndContext: SettingsWithContext) {
+  let response: ApiErrorResponse | undefined
+  try {
+    setTimeout(TimeoutForSavingFailures)
```

Comment:
there is no resolve function for this setTimeout ... can we remove it?

Reply:
It is actually a function of _useCutomFetch()_ that is imported [here](https://github.com/gobitfly/beaconchain/blob/c1e459f4398bc57dafcee09cea9b9c201025f89b/frontend/components/notifications/management/NotificationsManagementDashboards.vue#L44). We discussed it [here](https://github.com/gobitfly/beaconchain/pull/528#discussion_r1668652319).

Should I really remove it or just rename it to avoid confusion?